packages/core/src/browser/view-container.ts

```diff
@@ -1178,6 +1182,10 @@ export class ViewContainerLayout extends SplitLayout {
 
         // The update function is called on every animation frame until the predefined duration has elapsed.
         const updateFunc = (time: number) => {
+            if (!this.parent) {
```

Comment:
if a widget gets closed during the animation it will throw in `MessageLoop.sendMessage`